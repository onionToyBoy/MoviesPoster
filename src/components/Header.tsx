import React from 'react';
import { Link } from 'react-router-dom';

import styles from './Header.module.css';

export const Header = () => {
	return (
		<div className={styles.container}>
			<div className={styles.widthLimiter}>
				<div className={styles.logo}>ZombieFilms</div>
				<div className={styles.buttons}>
					<Link to='/home' className={styles.button}>
						Home
					</Link>
					<Link to='/about' className={styles.button}>
						About
					</Link>
				</div>
			</div>
		</div>
	);
};
