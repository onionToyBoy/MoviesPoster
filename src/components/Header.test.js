import React from 'react';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header test', () => {
	test('Should render Header component correct', () => {
		const component = shallow(<Header />);
		expect(component).toMatchSnapshot();
	});
});
