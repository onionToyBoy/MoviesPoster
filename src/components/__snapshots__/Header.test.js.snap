// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test Correct Header snapshot 1`] = `ShallowWrapper {}`;

exports[`Header test Should render Header component correct 1`] = `ShallowWrapper {}`;
